pages/_en-US/ds-index/forwarders.md

```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
 1. Scan this QR code to install the latest version of [Universal-Updater](https://github.com/Universal-Team/Universal-Updater)<br>
    ![Universal-Updater QR code](https://db.universal-team.net/assets/images/qr/universal-updater-cia.png)
 1. Open Universal Updater from your HOME Menu
-1. Install the NDSForwarder package
-1. NDSForwarder and its required files are now set up in its respective locations
-
-### Part 2: NDSForwarder
-1. Open the Homebrew Launcher
-1. In the Homebrew Launcher, open `NDS Forwarder Generator`
-1. Navigate to your game's location and press <kbd class="face">A</kbd>
+1. Install the YANBF package
+1. YANBF and its required files are now set up in its respective locations
+
+### Part 2: Generating forwarder titles
+1. Power off your console
+1. Insert your SD card into your computer
+1. Install [Python 3](https://python.org/downloads) for your computer
+1. Go to [YANBF releases page](https://github.com/lifehackerhansol/YANBF/releases) and download the respective `.zip` file for your computer's OS
+1. Extract the `.zip` file you downloaded
+1. Open a terminal (or Command Prompt), then `cd` to the `generator` folder
+1. .
+    - Windows users: run `pip install -r requirements.txt`
+    - Others: run `pip3 install -r requirements.txt`
+1. For macOS users:
+    1. Open the YANBF GUI `.dmg` file
+    1. Copy your app to the `generator` folder
+1. Double click on the GUI application to open the generator
```

Comment:
```suggestion
    1. Copy the app to your Applications folder
1. Open the GUI application
1. Set the `Generator Directory` on the right to the `generator` folder
```
The first change is fine either way I guess, installing to Applications makes more sense imo as that's more normal and the DMG even tells you to, but I guess either's fine.

Second line is just a bit oddly specific imo, it's not always even a double click to open an application.

The final change could probably be worded better, but at least on macOS you need to set that manually. Not sure if you need to on other OSes.